packages/roosterjs-editor-core/tsconfig.json

```diff
@@ -20,7 +20,10 @@
     "lib": [
       "es6",
       "dom"
-    ]
+    ],
+    "types": [
+      "@types/jasmine",
+    ],
```

Comment:
these tsconfig edits are required because by default tsc includes _all_ @types in node_modules.

This lead to a conflict, because this pr introduces jest (as our test runner for the puppeteer test). This change disables the automatic ambient inclusion of types and makes the inclusion of jasmine explicit.